modules/core/src/test/java/org/apache/ignite/client/SslParametersTest.java

```diff
@@ -271,58 +261,51 @@ public void testOneCommonProtocol() throws Exception {
-
-        for (int i = 0; i < n; i++) {
-            this.cipherSuites = cipherSuites != null && i < cipherSuites.length ? cipherSuites[i] : null;
-            this.protocols = protocols != null && i < protocols.length ? protocols[i] : null;
+    private void checkSuccessfulClientStart(String[] cipherSuites, String[] protocols) throws Exception {
+        this.cipherSuites = F.isEmpty(cipherSuites) ? null : cipherSuites;
+        this.protocols = F.isEmpty(protocols) ? null : protocols;
 
-            IgniteClient client = Ignition.startClient(getClientConfiguration());
+        IgniteClient client = Ignition.startClient(getClientConfiguration());
 
-            client.getOrCreateCache(TEST_CACHE_NAME);
+        client.getOrCreateCache(TEST_CACHE_NAME);
 
-            client.close();
-        }
+        client.close();
     }
 
     /**
      * @param cipherSuites list of cipher suites
      * @param protocols list of protocols
-     * @throws Exception If failed.
      */
-    private void checkClientStartFailure(String[][] cipherSuites, String[][] protocols) throws Exception {
-        checkClientStartFailure(cipherSuites, protocols, ClientConnectionException.class, "Ignite cluster is unavailable");
+    private void checkClientStartFailure(String[] cipherSuites, String[] protocols) {
+        checkClientStartFailure(
+            cipherSuites,
+            protocols,
+            ClientConnectionException.class,
+            "Ignite cluster is unavailable"
+        );
     }
 
     /**
      * @param cipherSuites list of cipher suites
      * @param protocols list of protocols
      * @param ex expected exception class
      * @param msg exception message
-     * @throws Exception If failed.
      */
-    private void checkClientStartFailure(String[][] cipherSuites, String[][] protocols, Class<? extends Throwable> ex, String msg) throws Exception {
-        int n = Math.max(
-            cipherSuites != null ? cipherSuites.length : 0,
-            protocols != null ? protocols.length : 0);
-
-        for (int i = 0; i < n; i++) {
-            this.cipherSuites = cipherSuites != null && i < cipherSuites.length ? cipherSuites[i] : null;
-            this.protocols = protocols != null && i < protocols.length ? protocols[i] : null;
-
-            int finalI = i;
-
-            GridTestUtils.assertThrows(null, new Callable<Object>() {
-                @Override public Object call() throws Exception {
-                    Ignition.startClient(getClientConfiguration());
-
-                    return null;
-                }
-            }, ex, msg);
-        }
+    private void checkClientStartFailure(
+        String[] cipherSuites,
+        String[] protocols,
+        Class<? extends Throwable> ex,
+        String msg
+    ) {
+        this.cipherSuites = F.isEmpty(cipherSuites) ? null : cipherSuites;
+        this.protocols = F.isEmpty(protocols) ? null : protocols;
+
+        GridTestUtils.assertThrows(null, new Callable<Object>() {
```

Comment:
Please pass every argument to single line.

Reply:
Done